compat/maven-artifact/src/main/java/org/apache/maven/artifact/repository/ArtifactRepositoryPolicy.java

```diff
@@ -132,18 +132,10 @@ public boolean checkOutOfDate(Date lastModified) {
 
     @Override
     public String toString() {
-        StringBuilder buffer = new StringBuilder(64);
-        buffer.append("{enabled=");
-        buffer.append(enabled);
-        buffer.append(", checksums=");
-        buffer.append(checksumPolicy);
-        buffer.append(", updates=");
-        buffer.append(updatePolicy);
-        buffer.append('}');
-        return buffer.toString();
+        return "{enabled=" + enabled + ", checksums=" + checksumPolicy + ", updates=" + updatePolicy + '}';
     }
 
-    public void merge(ArtifactRepositoryPolicy policy) {
```

Comment:
fluid api.

Reply:
not in this PR, maybe not at all

Reply:
kindly request your feedback on this.
- https://github.com/apache/maven/pull/2417